src/verilog.y

```diff
@@ -3412,6 +3412,7 @@ patternMemberOne<patmemberp>:	// IEEE: part of pattern and assignment_pattern
 	//			// From assignment_pattern_key
 	|	yDEFAULT ':' expr			{ $$ = new AstPatMember($1,$3,nullptr,nullptr); $$->isDefault(true); }
 	|	yDEFAULT ':' patternNoExpr		{ $$ = nullptr; BBUNSUP($2, "Unsupported: '{} .* patterns"); }
+	|       simple_type ':' expr			{ $$ = new AstPatMember($1->fileline(), $3, $1, nullptr); $$->isDefault(true); }
```

Comment:
Instead of here please try uncommenting this (so the grammar us closer to IEEEs)

//                      // IEEE: assignment_pattern_key
        //UNSUP simple_type                             { $1->v3error("Unsupported: '{} with data$

Reply:
Hmm, then my solution needs to be redone as we have no way of telling that the text is `simple_type` in the `patternMemberOne` rule.

Reply:
Ok, I missed that. vIf you follow the grammar and what you have supports the same inputs then we can do it your way, but we need to put a comment where the other simple type unsupported text is to instead say this us parsed in that other rule (versus unsupported which it no longer is).  Thanks.

Reply:
Ah, yeah. I understand. If it is cleaner to make the other way I can do it like that... I don't know if it will be that easily maintainable. I was wondering where to put this rule and found the way I did it more natural but maybe it's better to follow IEEE.

Reply:
Yes, generally we try to match IEEE unless doing so breaks something. Thanks.

Reply:
I think that solution is now cleaner. Thank you.